Enhancements/TokyoMirage_NegLod/rules.txt

```diff
@@ -0,0 +1,40 @@
+[Definition]
+titleIds = 00050000101ED700,00050000101ED800,0005000010131D00
+name = Negative texture LOD bias
+path = "Tokyo Mirage Sessions FE/Graphics/Negative Texture LOD" 
```

Comment:
Might be more fitting in Enhancements then Graphics, but I'll leave that up to you.